converter/convert.go

```diff
@@ -191,6 +194,15 @@ func makeManifests(b *bundle.Bundle, targetReference reference.Named, bundleConf
 	return manifests, nil
 }
 
+func makeSortedImages(images map[string]bundle.Image) []string {
```

Comment:
It is nitpicking but I found the name of this function is not very explicit. WDYT about something `sortImageNames`?

Reply:
The other functions folow the `make*` pattern: `makeDescriptor`, `makeManifests`, `makeAnnotations` so I guess it's logical to keep it as is.